quicktask/app/Http/Controllers/NewTaskController.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Task;
+use Illuminate\Http\Request;
+
+class NewTaskController extends Controller
```

Comment:
```suggestion
class TaskController extends Controller
```
Tất cả các action với task em sẽ viết ở đây.